.tekton/rhtap-consoledot-frontend-build-tests-pull-request.yaml

```diff
@@ -208,6 +208,21 @@ spec:
       workspaces:
       - name: source
         workspace: workspace
+    - name: inject-dockerfile
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/Shakedaviv50313/rhtap-consoledot-frontend-build/
```

Comment:
Change this to this repo before merging to main

Reply:
It seems making this change broke the tests. Try adding two new parameters: git-url and revision. And pass them to your task from the pipeline parameters called like that. So the values should be `$(params.git-url)` and `$(params.revision)`. Maybe doing this, we won't need to make these changes anymore to more tasks